samples/prot_test/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _prot_test:
+
+Protection tests
+#################################
+
+Overview
+********
+This test provides a set of shell commands to test
+protection against the following security issues:
+
+* Write to read-only data.
+* Write to text.
+* Execute from data.
+* Execute from stack.
+* Execute from heap.
+
+Messages that begin with FAIL: are indicative of a lack of
```

Comment:
@inakypg Should this be in a form that would make this test fit into the TCF infrastructure?